frontend/src/components/Preview/style.css

```
@@ -3,9 +3,25 @@
 pre {
     white-space: pre-wrap;
     word-wrap: break-word;
+    padding: 10px 0 0 10px;
+}
+
+.html-preview-iframe {
+    width: 100%;
+    height: 100%;
+    min-height: 500px;
+
+    border: none;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)